test/e2e/kubelet.go

```diff
@@ -130,10 +130,7 @@ func updateNodeLabels(c clientset.Interface, nodeNames sets.String, toAdd, toRem
 	}
 }
 
-// Calls startVolumeServer to create and run a nfs-server pod. Returns server pod and its
-// ip address.
-// Note: startVolumeServer() waits for the nfs-server pod to be Running and sleeps some
-//   so that the nfs server can start up.
+// Wraps startVolumeServer to create and run a nfs-server pod. Returns server pod and its ip address.
 func createNfsServerPod(c clientset.Interface, config VolumeTestConfig) (*v1.Pod, string) {
```

Comment:
there's theres `initNFSserverPod() *v1.Pod` in [persistent_volumes.go](https://github.com/jeffvance/kubernetes/blob/58cad044a57a8d0b63eb69122e35ac487e58e710/test/e2e/persistent_volumes.go#L90) so we've got a small overlap here.  Since we always need the NFS IP, `initNFSserverPod` could be replaced with `createNfsServerPod`